csm/conf/alerts_setup.yaml

```diff
@@ -0,0 +1,2 @@
+support_bundle:
+    - cortxcli csm_bundle_generate alerts
```

Comment:
This can be moved to csm/cli/conf folder

Reply:
Done